e2e/scenario/scenario_ubuntu2204-gpu-nodriver.go

```diff
@@ -0,0 +1,40 @@
+package scenario
+
+import (
+	"github.com/Azure/agentbaker/pkg/agent/datamodel"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute"
+)
+
+func ubuntu2204gpuNoDriver() *Scenario {
+	return &Scenario{
+		Name:        "ubuntu2204_gpuNoDriver",
```

Comment:
can we rename to something like "ubuntu2204-gpu-nodriver" to keep format consistent scenario name formatting? 

also open to suggestions for changing the naming format altogether

Reply:
yep, was not 100% sure what was desired, will adust